http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzureInterpreter.scala

```diff
@@ -0,0 +1,389 @@
+package org.broadinstitute.dsde.workbench
```

Comment:
maybe name the file `AzurePubsubHandlerInterp` or sth? `AzureInterpreter` seems too generic. Same for `AzureAlgebra`

Reply:
I'm just following the `util` package naming convention.

The corresponding GKE, VPC, and Gce equivalents are called `[X]Interpreter` and `[X]Algebra`. Are you suggestion we rename these too? I could definitely get behind that

Reply:
I think those are fine for now cuz they're not directly "messageHandlers"...If down the line, we make those more of `messageHandler`, then we can rename them...but yea, naming for some those files are a bit confusing as well